tower-load/src/peak_ewma.rs

```diff
@@ -120,12 +123,16 @@ where
                 );
                 Change::Insert(k, peak_ewma)
             }
+            Poll::Ready(Err(e)) => return Poll::Ready(Err(e)),
+            Poll::Pending => return Poll::Pending,
         };
 
-        Ok(Async::Ready(change))
+        Poll::Ready(Ok(change))
     }
 }
 
+impl<D, I> Unpin for PeakEwmaDiscover<D, I> {}
```

Comment:
Probably not okay

Reply:
This is safe since we have no unsafe projections.